test/blackbox-tests/test-cases/alias-multiple.t

```diff
@@ -0,0 +1,53 @@
+Testing multiple aliases in rules stanza
+
+First we start with a dune-project before alias was introduced:
+  $ cat > dune-project << EOF
+  > (lang dune 1.9)
+  > EOF
+
+  $ cat > dune << EOF
+  > (rule
+  > (alias a)
+  > (action (echo "I have run")))
+  > EOF
+
+  $ dune build @a
+  File "dune", line 2, characters 0-9:
+  2 | (alias a)
+      ^^^^^^^^^
+  Error: 'alias' is only available since version 2.0 of the dune language.
```

Comment:
```suggestion
  Error: 'alias' is only available since version 2.0 of the Dune language.
```

Reply:
This is the error message about the "dune lang" and is capitalized as such. Not in my control.